### ISSUE TITLE
Enhancement: Alert when rare poke's on nearby

### ISSUE THREAD

**Issue by npdoris** [PokeAlert/PokeAlert#149]
For those extra rare pokemon, it'd be nice to know when they show up on nearby for any particular scan, even if they never get scanned directly (if they are outside the scan area for instance). Even better would be to automatically trigger a scanner to hunt for that particular pokemon signature, ignoring any boundaries (within reason). This would help for those in the endgame with only a few accounts scanning and only a handful of pokemon that are worth the need of an alert. With only a notification, we could manually search around where it was seen on nearby.

And thanks a lot for this app, it has been incredibly helpful to find those shy pokemon :P

**Comment by TFD777** [comment]
Useless feature. It will slower down your regular scan. Just increase the area of ur scan and create more accounts.

**Comment by npdoris** [comment]
Well abrasive people skills aside, I still disagree that it is useless. Making new accounts is a non trivial act, especially when they keep getting banned, and is certainly something that Niantic doesn't want. Furthermore, the larger fetch area of the nearby let's you more efficiently search an area for what you really want, and  being able to search far from the boundary cuts back on searching dead space on an overly large search radius. Also, I have a lot of pokemon I want to be notified about, but only a handful that I'd really go above and beyond for, and I'd like those special few to be found quickly no matter where they are.

**Comment by halloonyc** [comment]
Just a thought here, but how would the "scanner accounts" know where the Pokémon is exactly if its just in their near by list? I'm no programmer or anything but I assume the Pokemons detected by the scanner account are pokemons that actually spawn for the scanner account as its roaming through an area. Once the pokemons spawn, it then somehow pins this on the map for you to see. Since near by pokemons haven't spawned yet, it would be pretty hard for you to track down. I've seen near by pokemons that are in the first slot of the tracker actually be a few hundred feet or more away. 

Creating more accounts / increasing your scan area would probably a lot more accurate

**Comment by npdoris** [comment]
I don't need to know exactly where it is at first, I just want to  know asap if the scanner gets even a whiff of a dragonite, lapras, etc. Have you ever had  some poke you want show up on the edge of your boundary with only a couple of minutes left? I have and would like to have known 10 minutes earlier that it had shown up on nearby (even if it didn't know the exact location). More scanning accounts is an answer, but not a good one in my opinion. I'd rather have the option to be more efficient with fewer accounts; to be able to add my brain to the searching instead of brute forcing a needlessly large area with a zillion accounts.

**Comment by michalbrz** [comment]
I agree with @npdoris - that would be really handy feature. And "creating more accounts" is not a solution - you can always say that, even for kilometer radius and 50 accounts. Moreover, that would be just more efficient, you just have to scan one more time to find that dragonite, instead of scanning bigger radius.
